### PR TITLE
Add SceJpeg, SceJpegArm and SceJpegEncArm

### PR DESCRIPTION
**Pull request by bythos14**
Full credit to @1whatleytay of the Vita3K Team for SceJpegUser code.

Permission to use the code was given here: Vita3K/Vita3K#978 
The code was taken from [SceJpegUser.cpp](https://github.com/Vita3K/Vita3K/blob/master/vita3k/modules/SceJpeg/SceJpegUser.cpp).

---

Full credit to @GrapheneCt for the SceJpegArm header

Permission to use the code was given here: GrapheneCt/Vita-PoC#1
Original header can be found at https://github.com/GrapheneCt/Vita-PoC. 

Removed error codes, which were [under suspicion of being copied](https://github.com/vitasdk/vita-headers/pull/473#issuecomment-696245981).

### CODE REVIEW

**Comment by bythos14** [comment]
Also added SceJpegEncArm library NIDs and a basic header for the library. The API is mostly identical to SceJpegEnc, just like how SceJpeg and SceJpegArm are very similar.

File: include/psp2/jpegarm.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+#define _PSP2_JPEGARM_H_
+
+#include <psp2/types.h>
+#include <psp2/jpeg.h>
+
+/**
+ * Error Codes
+ */
+typedef enum SceJpegArmErrorCode {
+    SCE_JPEG_ARM_OK = 0 // No error codes right now...
```
Comment (d3m3vilurr) [inline include/psp2/jpegarm.h:16]: could you remove this comment?

Comment (bythos14) [reply include/psp2/jpegarm.h]: Done

**Comment by d3m3vilurr** [comment]
merged ffda9f6 4d01fb0 5b1c104

**Pull request closed without merging**